lib/gssapi/krb5/gsskrb5_locl.h

```diff
@@ -83,13 +83,15 @@ typedef struct gsskrb5_ctx {
   struct krb5_ticket *ticket;
   time_t endtime;
   HEIMDAL_MUTEX ctx_id_mutex;
+  HEIMDAL_MUTEX *ctx_id_mutexp;
```

Comment:
Does this change belong in this commit?

Reply:
Oh, I took constification too far.  Security contexts should never be `const`.  I'll fix it.  Thanks for catching this.